Source/DIConnect/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -18,252 +18,252 @@ import { TFunction } from "i18next";
-                    </TooltipHost>
-                    {
-                        message.unknown &&
-                        <TooltipHost content="Unknown" calloutProps={{ gapSpace: 0 }}>
-                            <ExclamationCircleIcon xSpacing="both" className="unknown" outline />
-                            <span className="semiBold">{formatNumber(message.unknown)}</span>
+    private messageContent = (message: any) => {
+        return (
+            <Flex className="listContainer" vAlign="center" fill gap="gap.small">
+                <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} grow={1}>
+                    <Text
+                        className="title-content width"
+                        content={message.title}
+                        title={message.title} >
+                    </Text>
+                </Flex.Item>
+                <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }}>
+                    {this.renderSendingText(message)}
+                </Flex.Item>
+                <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} shrink={false}>
+                    <div>
+                        <TooltipHost content={this.props.t("TooltipSuccess")} calloutProps={{ gapSpace: 0 }}>
+                            <AcceptIcon xSpacing="after" className="succeeded" outline  />
+                            <span className="semiBold">{formatNumber(message.succeeded)}</span>
+                        </TooltipHost>
+                        <TooltipHost content={this.props.t("TooltipFailure")} calloutProps={{ gapSpace: 0 }}>
+                            <CloseIcon xSpacing="both" className="failed" outline />
+                            <span className="semiBold">{formatNumber(message.failed)}</span>
                         </TooltipHost>
-                    }
-            </div>
-        </Flex.Item>
-        <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} >
-          <Text
-            truncated
-            className="semiBold"
-            content={message.sentDate}
-          />
-        </Flex.Item>
-        <Flex.Item shrink={0}>
-          <Overflow message={message} title="" />
-        </Flex.Item>
-      </Flex>
-    );
-  }
+                        {
+                            message.unknown &&
+                            <TooltipHost content="Unknown" calloutProps={{ gapSpace: 0 }}>
+                                <ExclamationCircleIcon xSpacing="both" className="unknown" outline />
+                                <span className="semiBold">{formatNumber(message.unknown)}</span>
+                            </TooltipHost>
+                        }
+                    </div>
+                </Flex.Item>
+                <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }} >
+                    <Text
+                        truncated
+                        className="semiBold"
```

Comment:
Doesn’t Text support weight=semibold?

Reply:
Text supports weight="semibold", however here using custom css for text across page

Reply:
Why, though?

Reply:
removed it for Text element only. semiBold class is still used by few spans in the same file, so haven't removed it from scss.